src/services/helper/sessions.js

```diff
@@ -426,14 +432,23 @@ module.exports = class SessionsHelper {
 						}
 					})
 				)
+
+				await Promise.all(
+					sessionDetails[0].data.map(async (session) => {
+						const mentorName = await UserProfileHelper.details('', session.userId)
```

Comment:
@ankitpws  get all the users data in one API

Reply:
@rakeshSgr there is no API which will give me all the mentor details

Reply:
check this getAllAccountsDetail